builtins/amp-video.md

```diff
@@ -82,9 +83,10 @@ Similar to the `video` tag `controls` attribute - if present, the browser offers
 
 If present, will automatically loop the video back to the start upon reaching the end.
 
-**muted**
+**muted (deprecated)**
```

Comment:
`unmuted autoplay` isn't inherently bad UX. It depends on the use case. For instance, if the user lands on a page where the who point it watching a video (like YouTube) playing unmuted by default makes sense. Keeping this, but positioning it as a desktop only feature might be the right way to go.

Regarding iOS, one idea that comes to mind is similar to the bottom video in your demo where it's just for design. If the video is specified as muted, this could be an indicator that it should always just stay muted. By note exposing an unmute control, this might be the stance that iOS is taking. (Although in my use case, it wouldn't make sense to expose any controls.)